apps/dot_ext/forms.py

```diff
@@ -33,7 +34,7 @@ class Meta:
         fields = ('name',
                   'client_type',
                   'authorization_grant_type', 'redirect_uris',
-                  'logo_uri', 'policy_uri', 'tos_uri', 'contacts',
+                  'logo_uri', 'client_uri', 'policy_uri', 'tos_uri', 'contacts',
```

Comment:
should just call it `uri` or `website` or similar. `client_uri` results in calls like `application.client_uri` what's the `client`?

Reply:
OK. That sounds good for changing the name to make more sense.  Thinking website_uri ?